File: Python/1158.py
n = int(input())
for i in range(0,n):
    a = input().split()
    x,y=a
    x= int(x)
    y=int(y)
    soma = 0
    j = 1
    while j <= y:
        if x % 2 != 0:
            soma = soma + x
            
            x = x + 1
            j = j + 1
        if x % 2 == 0:
            x = x + 1
    print(soma)
            

File: CSharp/1158.cs
using System;
using System.Linq;

class URI
{
    static void Main()
    {
        int n = int.Parse(Console.ReadLine());
        int[] x;
        int soma=0;

        for(int i=0; i<n; i++)
        {
            x= Console.ReadLine().Split().Select(int.Parse).ToArray();
            if(x[0]%2!=0)
            {
                Console.WriteLine(x[1]*(x[0]+x[1]-1));
            }
            else{Console.WriteLine(x[1]*(x[0]+x[1]));}
        }
    }
}
